api_tests/institutions/views/test_institution_nodes_list.py

```diff
@@ -66,6 +67,16 @@ def test_does_not_return_private_nodes_with_auth(self, app, user, public_node, u
         assert user_private_node._id not in ids
         assert private_node._id not in ids
 
+    def test_registration_not_returned(self, app, institution, public_node, institution_node_url):
+        registration = RegistrationFactory(project=public_node, is_public=True)
```

Comment:
Does this test fail without your change? Doesn't the registration have to be affiliated with the institution?